packages/validator/src/cli-validator/run-validator.js

```diff
@@ -223,26 +217,20 @@ async function runValidator(cliArgs, parseOptions = {}) {
       continue;
     }
 
-    if (context.config.errorsOnly) {
-      logger.debug(
-        'Running in "errors only" mode - only error-severity validations will be displayed.'
-      );
-    }
-
     // Check to see if we should be passing back a non-zero exit code.
     if (results.error.summary.total) {
       // If we have any errors, then exit code 1 is returned.
       exitCode = 1;
-    } else if (!context.config.errorsOnly) {
-      // If the # of warnings exceeded the warnings limit, then this is an error.
-      const numWarnings = results.warning.summary.total;
-      const warningsLimit = context.config.limits.warnings;
-      if (warningsLimit >= 0 && numWarnings > warningsLimit) {
-        exitCode = 1;
-        logger.error(
-          `Number of warnings (${numWarnings}) exceeds warnings limit (${warningsLimit}).`
-        );
-      }
+    }
+
+    // If the # of warnings exceeded the warnings limit, then this is an error.
+    const numWarnings = results.warning.summary.total;
+    const warningsLimit = context.config.limits.warnings;
+    if (warningsLimit >= 0 && numWarnings > warningsLimit) {
+      exitCode = 1;
+      logger.error(
+        `Number of warnings (${numWarnings}) exceeds warnings limit (${warningsLimit}).`
+      );
```

Comment:
This can be independent of "errors only" now that warnings won't even be in the results for that case